### DISCUSSION TITLE
Is it possible to add a Browser-GUI?

### DISCUSSION THREAD

**Discussion by Moosbueffel** [cyanfish/naps2#367] [Development]
I use naps2 for a short time, since I installed paperless-ngx on my Raspberry. This works well so far. But it would quite better, if I can install naps on my headless Raspi and control it via http and a browser. This will also provide to use it easier from other people of my household and save me the time to transfer files from Windows to my Raspi.

I have seen, that you now have a possibility to share naps. Also there is a cli. So perhaps naps is developed as MVC and the GUI is seperated from the rest. In this case perhaps is a chance to install it as a server. In best condition as a docker file.

Thanks for your great work!

**Comment by cyanfish** [comment]
Have a look at #120
